config/application.rb

```diff
@@ -24,6 +24,8 @@ class Application < Rails::Application
     config.active_record.default_timezone = :local
     config.active_record.time_zone_aware_attributes = false
 
+    config.active_storage[:variable_content_types] << "image/webp"
```

Comment:
Can we move it into an initializer please?

Reply:
done